src/api.js

```diff
@@ -0,0 +1,12 @@
+const BASE_URL = 'https://bootcamp-api.codeit.kr/api/sample/'
+
+export async function getUserInfo(path) {
+  const response = await fetch(BASE_URL + path);
+
+  if (!response.ok) {
+    throw new Error("유저 정보를 불러오는데 실패했습니다.")
+  }
+
+  const body = await response.json();
```

Comment:
이미 위에서 fetch 한 부분을 await 처리해줬기 때문에 이 부분은 await 가 의미가 없을 거 같아요!
```suggestion
  const body = response.json();
```